addons/hr/static/src/components/avatar_card/avatar_card_popover_patch.xml

```diff
@@ -4,6 +4,14 @@
         <xpath expr="//div[hasclass('o_avatar_card_buttons')]" position="inside">
             <button class="btn btn-secondary btn-sm" t-if="user.employee_ids?.length > 0" t-on-click.stop="onClickViewEmployee">View profile</button>
         </xpath>
+        <xpath expr="//a[hasclass('o-mail-avatar-card-tel')]" position="after">
+            <span t-if="user.work_location_name" class="text-muted" data-tooltip="Work Location">
```

Comment:
I'm wondering, is `data-tooltip` attribute automatically marked for translation?

Reply:
https://github.com/odoo/odoo/issues/160040